internal/command/local/override/command.go

```diff
@@ -13,24 +13,22 @@ func New(local *config.Local) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "override --sandbox=<sandbox> [--workload=<workload>] --workload-port=<port> --with=<target> [--except-status=...] [--detach]",
 		Short: "Override sandbox HTTP traffic using a local service",
-		Long: `Using the 'override' command, when a request comes into the sandbox it is
-delivered to the local service. The response of the local service determines
-whether or not the request will subsequently be delivered to its original
-destination (i.e. the sandbox workload).
-
-When the request is not subsequently delivered to the original destination,
-the response from the local service is the response returned to the client.
-
-When the local service is not running, requests will be delivered to the
-original destination after failing to communicate with the local service.
-
-By default, overrides apply when the response from the local service
-includes the header 'sd-override: true'.
-
-You can use the '--except-status' flag to specify HTTP response codes that
-should not be overridden. When set, all other traffic will be overridden
-except for the specified status codes, which will fall through to the
-original sandboxed destination.`,
+		Long: `The 'override' command lets you intercept both HTTP and gRPC traffic coming into your sandbox and process it with a local service you specify (such as on your laptop).
+
+How it works:
+- Any HTTP or gRPC request to your sandbox is first routed to your local service.
+- Your local service processes the request and replies.
+- By default:
+    * If your local service's response contains the header 'sd-override: true' (for HTTP), or the metadata 'sd-override: true' (for gRPC), the response from your local service is immediately sent back to the client—the request does NOT reach the original sandbox workload.
```

Comment:
> or the metadata 'sd-override: true' (for gRPC)

I have not tested this.  Looked at the implementation, and it looks like it would likely work, but it would go through several implicit coding steps under the hood:  gRPC metadata -> http2 headers -> plain Go http headers (using http2 transport on top of http).

I think @daniel-de-vera tested this?

Reply:
I removed the reference to metadata, @daniel-de-vera  tells me it's untested and we can leave it as "set a response header" instead for both HTTP and gRPC.